pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/FunctionsBase.java

```diff
@@ -288,14 +289,4 @@ public Response downloadFunction(final @QueryParam("path") String path) {
     public List<ConnectorDefinition> getConnectorsList() throws IOException {
         return functions.getListOfConnectors();
     }
-
-    @GET
```

Comment:
we will also need change here: https://github.com/apache/incubator-pulsar/blob/master/pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/FunctionsImpl.java#L278

Reply:
good catch! Didn't even realize we had a CLI options get this info